extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js

```diff
@@ -4192,6 +4210,9 @@ var globalRequire = require;
 					var adErrorEvent = google.ima.AdErrorEvent.Type;
 					_IMA_SDK_destroyAdsLoader();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the declaration of "_IMA_SDK_destroyAdsLoader" before this usage. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)